docs/index.md

```diff
@@ -59,107 +55,94 @@ reference/index
 
 ```{toctree}
 :caption: Community
-:name: mastertoc
 :maxdepth: 1
 :hidden:
 
-integrations/index
 community/discord/index
 community/github/index
 community/contribution_guidelines/index
 ```
 
 <h1 align="center" weight='300' style="color: var(--sd-color-card-text);" >The testing framework dedicated to  ML models, from tabular to LLMs</h1>
-<p align="center"> 
-   <a href="https://www.giskard.ai/knowledge-categories/blog/?utm_source=github&utm_medium=github&utm_campaign=github_readme&utm_id=readmeblog"><b>Blog</b></a> &bull;  
+<p align="center">
+   <a href="https://www.giskard.ai/knowledge-categories/blog/?utm_source=github&utm_medium=github&utm_campaign=github_readme&utm_id=readmeblog"><b>Blog</b></a> &bull;
   <a href="https://www.giskard.ai/?utm_source=github&utm_medium=github&utm_campaign=github_readme&utm_id=readmeblog"><b>Website</b></a> &bull;
   <a href="https://gisk.ar/discord"><b>Discord</b></a>
  </p>
 <br />
 
 
-::::::{grid} 1 1 3 3
-:child-align:
-:gutter: 1
-
-:::::{grid-item}
-:child-align:
-::::{grid} 1 1 1 1
-:child-align:
+:::::{grid} 1 1 3 3
 :gutter: 1
 
+::::{grid-item}
 :::{card} <h2><center> Getting Started </center></h2>
 :margin: 1
 :width: 100%
-:text-align:
-:link: getting_started/index.html
+:text-align: center
+:link: getting_started/index.md
 :img-bottom: assets/intro/coffee.png
 :::
+::::
 
+::::{grid-item}
 :::{card} <h2><center> Tutorials </center></h2>
 :margin: 1
 :width: 100%
-:text-align:
-:link: tutorials/index.html
+:text-align: center
+:link: tutorials/index.md
 :img-bottom: assets/intro/test_turtle.png
 :::
+::::
 
+::::{grid-item}
 :::{card} <h2><center> API Reference </center></h2>
 :margin: 1
 :width: 100%
-:link: reference/index.html
+:link: reference/index.rst
 :img-bottom: assets/intro/Giskard_Turtle_Computer.png
 :::
 ::::
 
-:::::
-
-:::::{grid-item}
-:child-align:
-::::{grid} 1 1 1 1
-:child-align:
-:gutter: 1
```

Comment:
one of these `child-align` was responsible for better alignment of cells. 

our public doc:
<img width="610" alt="Screenshot 2023-11-24 at 11 54 55" src="https://github.com/Giskard-AI/giskard/assets/32709181/1390d522-727f-4e18-9c9b-9c0c9af4de1d">

doc in this pr:
<img width="776" alt="Screenshot 2023-11-24 at 11 56 51" src="https://github.com/Giskard-AI/giskard/assets/32709181/ea77a60c-6f13-40b0-a8cf-90171d031932">

Reply:
> one of these `child-align` was responsible for better alignment of cells.
> 
> our public doc: <img alt="Screenshot 2023-11-24 at 11 54 55" width="610" src="https://user-images.githubusercontent.com/32709181/285437029-1390d522-727f-4e18-9c9b-9c0c9af4de1d.png">
> 
> doc in this pr: <img alt="Screenshot 2023-11-24 at 11 56 51" width="776" src="https://user-images.githubusercontent.com/32709181/285437212-ea77a60c-6f13-40b0-a8cf-90171d031932.png">

there were mistakes in it. Plus we should not nest grid. If it's only the order, I can change it back or find some stuff

Reply:
Why we should not nest grid? I used the following doc: https://sphinx-design.readthedocs.io/en/furo-theme/grids.html#nesting-grids
Yes I think the order is important here, can we please revert this back?